src/mmwave/model/antenna-array-model.cc

```diff
@@ -80,6 +82,22 @@ AntennaArrayModel::GetTypeId ()
                    BooleanValue (true),
                    MakeBooleanAccessor (&AntennaArrayModel::m_isotropicElement),
                    MakeBooleanChecker ())
+    .AddAttribute ("AntennaElementPattern",
+                   "The available antenna element patterns refer to '3GPP-MmWave', '3GPP-V2V'",
+                   StringValue ("3GPP-MmWave"),
+                   MakeStringAccessor (&AntennaArrayModel ::m_antennaElementPattern),
+                   MakeStringChecker ())
+    .AddAttribute ("AntennaElements",
+                   "The number of antenna elements",
+                   UintegerValue (4),
+                   MakeUintegerAccessor (&AntennaArrayModel::SetTotNoArrayElements,
+                                         &AntennaArrayModel::GetTotNoArrayElements),
+                   MakeUintegerChecker<uint64_t> ())
+    .AddAttribute ("NumSectors",
```

Comment:
You also have this attribute in the helper. I would recommend having it in a single place!